internal/store/driver.go

```diff
@@ -382,7 +382,7 @@ func (d *Driver) calculateHTTPSEdges() []ingressv1alpha1.HTTPSEdge {
 			modSet, err := d.getNgrokModuleSetForIngress(ingress)
 			if err != nil {
 				d.log.Error(err, "error getting ngrok moduleset for ingress", "ingress", ingress)
-				continue
+				panic(err)
```

Comment:
This sounds too invasive to me, especially since it is bringing all tunnels down if the pods are crashlooping?

Reply:
@bobzilladev hmm, I don't think so because we wouldn't get to the part where it even [gets to update the tunnels](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/internal/store/driver.go#L276)? If I'm understanding it correctly...